.changeset/plenty-apples-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/localize': patch
```

Comment:
Does this need to change?

Reply:
Yeah, should be `@lion/ui` indeed, with maybe `[localize]` in front of the description